### ISSUE TITLE
Show confirmation and refresh source control panel on Changelist submitted

### ISSUE THREAD

**Issue by Fahien** [stef-levesque/vscode-perforce#73]
Currently, the only way to see that a changelist has been correctly submitted is by checking the **Perforce Logs**. It would be useful some kind of _visual feedback_ according to the result of the submit command.

Moreover, a just submitted changelist still appear in the source control panel, even clicking the **refresh** button. To get rid of that you have to _check out_ a file for editing or reload the whole vscode window.

**Comment by stef-levesque** [comment]
The confirmation feedback make absolute sense, I'll add it to the next release.

Although, I couldn't repro the ghost-changelist after submitting. If you can easily repro it, could you include the Perforce Logs in this issue?

**Comment by Fahien** [comment]
```
p4.exe submit -c 4714271
Submitting change 4714271.
Locking 1 files ...
edit //MyProject/DEV/FAHIEN/myfile.py#2
Change 4714271 renamed change 4714274 and submitted.
p4.exe login -s
p4.exe info 
p4.exe login -s
p4.exe changes -c fahien_client -s pending
p4.exe opened
```

**Comment by stef-levesque** [comment]
Now fix in version 2.2.0, please update.
